TrackReco/MakeHits_module.cc

```diff
@@ -167,22 +166,25 @@ namespace emph {
   
   void MakeHits::beginRun(art::Run& run)
   {
-    art::ServiceHandle<emph::cmap::ChannelMapService> fChannelMap;
     art::ServiceHandle<emph::geo::GeometryService> geo;
     auto emgeo = geo->Geo();
+    nPlanes = emgeo->NSSDPlanes();
+    /*
```

Comment:
Is there a reason to keep this code around?

Reply:
Nope, I just forgot about it.  I've removed it.